### PR TITLE
19 revise get system status endpoint

### LINKED ISSUE #19: Revise get system status endpoint
Revise it based on the most common health check convention

### PR DESCRIPTION
**Pull request by basarbyz**
Closes #19

### CODE REVIEW

File: tests/test_crud.py [hunk 1]
```diff
@@ -175,3 +175,26 @@ def test_delete(self):
...
+    def test_get_version_info(self):
+        response = response = requests.request("GET", settings.GATEWAY_URL+"/system/version")
+        self.assertEqual(response.status_code, 200)
+        data = response.json()
+        self.assertIn("application_version", data)
+        self.assertIn("dependencies", data)
+        dependencies = ["fastapi", "aiohttp", "asyncpg", "pydantic", "redis", "uvicorn"]
+        for dep in dependencies:
+            self.assertIn(dep, data["dependencies"])
+            self.assertEqual(data["dependencies"][dep], importlib.metadata.version(dep))
```
Comment (djs0109) [inline tests/test_crud.py:200]: `importlib` is not imported

**Review by djs0109 — CHANGES_REQUESTED** [review]

File: tests/test_crud.py (hunk 1 above)
Comment (djs0109) [reply tests/test_crud.py:201]: fixed

File: backend/api/main.py [hunk 2]
```diff
@@ -427,48 +431,90 @@ async def get_match_status(
     description="Get the status of the system. This is to allow the frontend to check whether the system is running properly.",
 )
 async def get_status():
-    system_status = {
+    checks = {
```
Comment (djs0109) [inline backend/api/main.py:434]: After changing the response model, the GUI also need to be adapted

Comment (djs0109) [reply backend/api/main.py:434]: fixed

**Comment by djs0109** [comment]
@basarbyz only FYI, there were some tiny points still missing. But now I have fixed them

**Review by djs0109 — APPROVED** [review]

**Comment by basarbyz** [comment]
okay, thank you

**Pull request merged**